### ISSUE TITLE
Shorthand for 'tmux send-keys'

### ISSUE THREAD

**Issue by thetredev** [thetredev/steamcmd#54]
`tmux` doesn't provide a builtin way to create new commands, for example via `tmux.conf`. I will have to check out tmux plugins for that. What I want to do is:

```
# tmux.conf

enter: send-keys "<args>" "Enter"
```

Which in turn would allow for:
```
tmus -S <socket> enter "<srcds/hlds command>"
```

**Comment by thetredev** [comment]
A unix socket would be the best solution I think.

**Comment by thetredev** [comment]
Fixed in 85cfdf4f1f0bda286abf324d8aa2ad2f625c9eba.
